recipes/basic_project_setup.ipynb

```diff
@@ -0,0 +1,421 @@
+   "source": [
+    "Now, let's set up variables needed to create an instance of the Kili object.\n",
+    "We'll need your API key and Kili's API endpoint.\n",
+    "If you're unsure how to look up your API key, refer to https://docs.kili-technology.com/docs/creating-an-api-key."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "KILI_API_KEY = getpass.getpass(\"Please enter your API key: \")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With variables set up, we can now create an instance of the Kili object."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "kili = Kili(api_key=KILI_API_KEY)\n",
+    "# Note: if you are using the on-premise version of Kili, you can specify the api endpoint here."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"create-project\"></a>\n",
+    "## Creating a basic Kili project"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To create a Kili project, you must first set up its interface.\n",
+    "We'll create a simple image project with just one simple classification job and two categories: `OBJECT_A` and `OBJECT_B`.\n",
+    "\n",
+    "To learn more about Kili project interfaces, refer to https://docs.kili-technology.com/docs/customizing-project-interface"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "interface = {\n",
+    "  \"jobs\": {\n",
+    "    \"JOB_0\": {\n",
+    "      \"mlTask\": \"CLASSIFICATION\",\n",
+    "      \"required\": 1,\n",
+    "      \"content\": {\n",
+    "        \"categories\": {\n",
+    "          \"OBJECT_A\": {\n",
+    "            \"name\": \"Object A\"\n",
+    "          },\n",
+    "          \"OBJECT_B\": {\n",
+    "            \"name\": \"Object B\"\n",
+    "          }\n",
+    "        },\n",
+    "        \"input\": \"radio\"\n",
+    "      }\n",
+    "    }\n",
+    "  }\n",
+    "}\n",
+    "\n",
+    "result = kili.create_project(\n",
+    "    title='Test Project',\n",
+    "    description='Project Description',\n",
+    "    input_type='IMAGE',\n",
+    "    json_interface=interface\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For further processing, we'll need to find out what our project ID is. We can easily retrieve it from the project creation response message."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "project_id = result['id']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, let's add some assets to be labeled. Since this is just an example, let's use some free off-the-shelf examples from the Internet."
```

Comment:
I've just realized I messed up this sentence. PLs change it to:

"We'll use some free off-the-shelf examples from the Internet"

Reply:
done